### DISCUSSION TITLE
Creating Parameters in query

### DISCUSSION THREAD

**Discussion by Matini97** [Budibase/budibase#2326] [Help] [answered]
Hello,
I'm currently learning SQL programming and using budibase as a medium to use different DBs and to get acquainted to handlebars.
When using Query PostgreSQL the Add Parameters button/setting appears to create a parameter, yet it can not be used in the query field unless I define it again, so my question would be, do u intent to make it possible to use the field to implement the parameter 
into the query directly or might it be that I'm just missing a mayor point here?

Tia
![Parameters](https://user-images.githubusercontent.com/88765407/128985113-68639afc-87f9-44a4-a090-e2686e72279d.PNG)

**Comment by mjashanks** [comment, marked as answer]
Hey @Matini97.

This should work ok. I just tested it with postgres, using the following setup:

![image](https://user-images.githubusercontent.com/3524181/128998262-cf452a1b-a48a-426f-acc4-c2e7b1ff60e0.png)

Let me know if I've misunderstood your issue :)

**Reply by Matini97** [reply]
worked fine seems i was just to dense -.-'

**Reply by Matini97** [reply]
@mjashanks  oh another question i got is, is there a way to set the type of the parameter in the field or not/not yet

**Reply by mjashanks** [reply]
Type as in `number`, `date` etc? No, not yet. We will be looking into improvements to this, but not until October at least.
